.github/workflows/e2e-testbench.yaml

```diff
@@ -0,0 +1,69 @@
+name: Run E2E Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      maxTestDuration:
+        description: 'Test duration (Eg: PT2H, P3D)'
+        required: true
+        default: 'PT1D'
+        type: string
+      generation:
+        description: 'Specifies the generation template which should be used by the testbench run'
+        required: false
+        default: 'Zeebe SNAPSHOT'
+        type: string
+      branch:
+        description: 'Specifies the branch, for which the E2E tests should be executed'
+        default: 'main'
+        required: false
+  schedule:
+    # Run at 7:00 on every monday
+    - cron:  '0 7 * * 1'
+env:
+  BUILD_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+
+jobs:
+  e2e-testbench:
+    name: E2E Testbench run
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: "${{ github.event.inputs.branch || 'main' }}"
+
+      - name: Build and push docker
+        id: build-docker
+        uses: ./.github/actions/build-push-docker-gcr
+        with:
+          branch: ${{ github.event.inputs.branch }}
+          secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secret_vault_address: ${{ secrets.VAULT_ADDR }}
+          secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
+
+      - name: Import Secrets
+        id: secrets
+        uses: hashicorp/vault-action@v2.4.2
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/data/products/zeebe/ci/zeebe TESTBENCH_PROD_CLIENT_SECRET;
+            secret/data/products/zeebe/ci/zeebe TESTBENCH_PROD_CONTACT_POINT;
+        # Start e2e test instance, do not wait for the result
+      - name: Run E2E Tests
+        run: .ci/scripts/distribution/e2e-testbench.sh
```

Comment:
:question: Is there any benefit from running this as a script and not just inline here? e.g.

```yaml
      - name: Run E2E Tests
        run: |
          chmod +x clients/go/cmd/zbctl/dist/zbctl
          read -r -d '' VARIABLES << EOM
          {
            "zeebeImage":"${{ steps.build-docker.outputs.image }}",
            "generationTemplate":"${{ github.event.inputs.generation || 'Zeebe SNAPSHOT' }}",
            "channel":"Internal Dev",
            "clusterPlan":"Production - M",
            "region":"new chaos",
            "properties":[
                "allInstancesAreCompleted"
            ],
            "testProcessId":"e2e-test",
            "branch":"${{ github.event.inputs.branch }}",
            "build":"${{ env.BUILD_URL }}",
            "testParams":{
                "maxTestDuration":"${{ github.event.inputs.maxTestDuration || 'PT5D' }}",
                "starter":[
                  {
                      "rate":50,
                      "processId":"one-task-one-timer"
                  }
                ],
                "verifier":{
                  "maxInstanceDuration":"15m"
                }
            }
          }
          EOM

          clients/go/cmd/zbctl/dist/zbctl create instance e2e_testbench_protocol --variables "${VARIABLES}"
```

Although, now I wonder if we just shouldn't have a `zbctl` action anyway.

Reply:
We have one :sweat_smile: https://github.com/zeebe-io/zbctl-action/

Reply:
What you proposed did not work. The script failed, but it didn't say why.  So I have just inlined the script here.

Reply:
> We have one sweat_smile https://github.com/zeebe-io/zbctl-action/

Nice. Totaly forgot about it. For now I wont use it, as zbctl is already available without the action.